tests/__init__.py

```diff
@@ -39,13 +39,37 @@ def set_up_test_graphs():
     #          \ /
     #           4
     non_trivial_graph = create_graph_helper([(0, 1), (1, 2), (1, 4), (2, 3), (3, 4)])
+    non_trivial_graph.name = 'non_trivial_graph'
 
     # Instantiate the non-trivial graph's complement
     # non_trivial_graph_complement =
     #               2
     #              / \
     #     1 - 3 - 0 - 4
     non_trivial_graph_complement = create_graph_helper([(2, 0), (3, 0), (4, 0), (3, 1), (4, 2)])
+    non_trivial_graph_complement.name = 'non_trivial_graph_complement'
+
+    # Instantiate some isomorphic graphs
+    # non_trivial_graph_isomorphism =
+    #           4
+    #          / \
+    #     0 - 1   3
+    #          \ /
+    #           2
+    non_trivial_graph_isomorphism = create_graph_helper([(0, 1), (1, 2), (2, 3), (3, 4), (1, 4)])
+    non_trivial_graph_isomorphism.name = 'non_trivial_graph_isomorphism'
+
+    isomorphic_graphs = [non_trivial_graph, non_trivial_graph_isomorphism]
```

Comment:
TODO: 3 isomorfismen in deze lijst stoppen, zodat:

1. `0` en `1`: isomorf
1. `0` en `2`: isomorf, dus `1` en `2` ook isomorf
1. `1` en `2`: reeds bekend dat isomorf, dus scheelt rekenwerk!